content/pt-br/docs/reference/glossary/event.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Evento
+id: event
+date: 2022-01-16
+full_link: /docs/reference/kubernetes-api/cluster-resources/event-v1/
+short_description: >
+   Um relatório de um evento em algum lugar do cluster. Geralmente denota alguma mudança de estado no sistema.
+
+aka: 
+tags:
+- core-object
+- fundamental
+---
+Cada evento é uma informação de um acontecimento em algum lugar do {{< glossary_tooltip text="cluster" term_id="cluster" >}}. 
+Geralmente denota alguma mudança de estado no sistema.
+
+<!--more-->
+
+Os eventos tem um tempo restrito de retenção, e os gatilhos e as mensagens podem estar envolvidos com tempo. 
```

Comment:
Minor:
* Perhaps "limitado" instead of "restrito" might be more idiomatic?
* The original context was "may evolve", not "may involve". I think a more faithful translation would be "podem evoluir com o tempo".